csg.js

```diff
@@ -6481,6 +6531,11 @@ for solid CAD anyway.
                     }
                     thisside = sideTagToSideMap[nextsidetag];
                 } // inner loop
+                // due to the logic of CAG.fromPoints()
+                // move the first point to the last
+                if (connectedVertexPoints.length > 0) {
```

Comment:
Actually, it corrects the order of the sides (points).

I think this is really important a when extruding CAGs that are rotating, morphing, etc.